colossalai/inference/core/config.py

```diff
@@ -34,21 +35,18 @@ class InferenceConfig:
     max_output_len: int = 256
     max_input_len: int = 256
     block_size: int = 16
-    gpu_utilization_rate: float = 0.7
     dtype: Union[str, torch.dtype] = torch.float32
     tp_size: int = 1
     pp_size: int = 1
     max_seq_len: Optional[int] = None
     quant_mode: Optional[str] = None
     revision: Optional[str] = None
+    # TODO: beam search is not support for now
+    beam_width: int = 1
 
     def __post_init__(self):
         self._verify_args()
 
     def _verify_args(self):
-        if self.gpu_utilization_rate > 1.0:
-            raise ValueError(
-                f"GPU utilization should be less than 1.0, but is set to {self.gpu_memory_utilization}."
-            )
```

Comment:
Why delete this?

Reply:
We're not going to use GPU Utilization rate at this moment as the allocation of physical cache does not depend on hardware evaluation (instead, we have hard limits on maximum batch size, etc).